ubuntudesign/www.canonical.com
# Remove YUI: Replace global nav functions

**Review and merge #105 first**

- Use the updated global nav JS & CSS
- Create one function for all (both) `toggle-menu` buttons
- Massively simplify and replace global nav javascript

QA
--

Check the global nav is at the top of the page on large screens and is in the footer on small screens.

Check it works the same as live in both cases.
